spec/models/user_spec.rb

```diff
@@ -4,6 +4,12 @@
 MAX_GOOD_PASSWORD = '!1aZ' * 32
 
 describe User do
+  describe 'Associations' do
+    it { should have_many(:authorizations) }
```

Comment:
@sbc100 is right. We should be using the `expect` syntax, which `shoulda-matchers` supports:
```ruby
it { is_expected.to have_many(:authorizations) }
```
This is the syntax we have been using in all the other specs that use `shoulda-matchers`.